gui-apps/waybar/waybar-0.11.0.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="Highly customizable Wayland bar for Sway and Wlroots based compositors"
+HOMEPAGE="https://github.com/Alexays/Waybar"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Alexays/${PN^}.git"
+else
+	SRC_URI="https://github.com/Alexays/${PN^}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="amd64 ~arm64"
```

Comment:
`ekeyword ~all waybar-0.11.0.ebuild`

Reply:
okay. thanks for pointing that out.

Reply:
Can you try to squash this commit with the one adding 0.11.0?  No point in adding it in 2 commits

Reply:
okay